scripts/setup.sh

```diff
@@ -30,6 +30,8 @@ if [ ! -d "/home/myproxy" ] ; then
   cp ./scripts/post-receive /home/myproxy/.scripts/post-receive
   cp ./scripts/pre-receive /home/myproxy/.scripts/pre-receive
   cp ./scripts/gitignore /home/myproxy/.scripts/.gitignore
+  # Disable SSH MOTD message for git user
+  touch /home/git/.hushlogin
```

Comment:
This disables the MOTD sent from the server when someone tries to SSH in. For example:

```bash
$ ssh git@challenges.ggwadera.xyz
PTY allocation request failed on channel 0
Welcome to Ubuntu 20.04.1 LTS (GNU/Linux 5.4.0-42-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Tue Sep  1 13:15:29 UTC 2020

  System load:  0.72               Users logged in:          1
  Usage of /:   18.9% of 24.06GB   IPv4 address for docker0: 172.17.0.1
  Memory usage: 48%                IPv4 address for eth0:    157.245.253.104
  Swap usage:   0%                 IPv4 address for eth0:    10.10.0.5
  Processes:    130

13 updates can be installed immediately.
0 of these updates are security updates.
To see these additional updates run: apt list --upgradable


fatal: Interactive git shell is not enabled.
hint: ~/git-shell-commands should exist and have read and execute access.
Connection to challenges.ggwadera.xyz closed.
```

Becomes: 

```bash
$ ssh git@challenges.ggwadera.xyz
PTY allocation request failed on channel 0
fatal: Interactive git shell is not enabled.
hint: ~/git-shell-commands should exist and have read and execute access.
Connection to challenges.ggwadera.xyz closed.
```